src/context.h

```diff
@@ -51,6 +52,8 @@ struct enftun_context
     struct enftun_dhcp dhcp;
     struct enftun_ndp ndp;
 
+    struct enftun_conn_state conn_state;
```

Comment:
Put this above tls and tun. Doesn't matter functionally, but conceptually it gets used before any of those.